src/pilot/templates/neutron-sriov.yaml

```diff
@@ -0,0 +1,20 @@
+## A Heat environment that can be used to deploy SR-IOV
+resource_registry:
+  OS::TripleO::Services::NeutronSriovAgent: ./overcloud/docker/services/neutron-sriov-agent.yaml
+
+parameter_defaults:
+  NeutronMechanismDrivers: ['sriovnicswitch', 'openvswitch']
+  NovaSchedulerDefaultFilters: ['RetryFilter','AvailabilityZoneFilter','RamFilter','ComputeFilter','ComputeCapabilitiesFilter','ImagePropertiesFilter','ServerGroupAntiAffinityFilter','ServerGroupAffinityFilter','PciPassthroughFilter']
+  NovaSchedulerAvailableFilters: ["nova.scheduler.filters.all_filters","nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter"]
+  HugePages: "default_hugepagesz=1GB hugepagesz=1G hugepages=112 iommu=pt intel_iommu=on"
```

Comment:
Is this not going to clash with the HugePages settings already set in dell-environment.yaml ?